src/lib.rs

```diff
@@ -840,6 +844,7 @@ pub struct Closed01<F>(pub F);
 /// The standard RNG. This is designed to be efficient on the current
 /// platform.
 #[derive(Copy, Clone, Debug)]
+#[cfg_attr(feature="serde-1",derive(Serialize,Deserialize))]
 pub struct StdRng {
```

Comment:
I personally think `StdRng` should not be serializable — we wish to be able to replace the algorithm in the future without breaking code, and if someone is serializing the RNG it implies they wish output to remain the same. `StdRng` is also architecture dependent (serialize on a 32-bit machine, deserialize on 64-bit — whoops). Users may use `IsaacRng` or the 64-bit version directly.

Reply:
That's a fair point.